bess-upf/values.yaml

```diff
@@ -136,6 +136,32 @@ config:
             burst_duration_ms: 10
             priority: 7
 
+# PFCP raw dump configuration
```

Comment:
Does this have to be its own section? Given that this is a configuration (config) section, can it be right between `coreDump` and `gratuitousArp`?